examples/online_coding/config/qwen2.5-1.5b-ppo.yaml

```diff
@@ -0,0 +1,298 @@
+  checkpoint_load_path: "/mnt/public/mg_ckpts/Qwen2.5-Coder-1.5B-tp2-pp1"
+  # checkpoint_load_path: None
+
+  offload_optimizer: True
+  offload_weight: True
+  offload_grad: True
+
+  enable_dp_load_balance: False
+
+  calculate_flops: False
+
+  seed: 1234
+
+  model:
+    precision: fp16
+    add_bias_linear: False
+
+    tensor_model_parallel_size: 2
+    pipeline_model_parallel_size: 1
+
+    activation: swiglu
+    sequence_parallel: True
+    # recompute_method: block
+    # recompute_granularity: selective
+
+    recompute_method: block
+    recompute_granularity: full
+    recompute_num_layers: 20
+
+    seq_length: ${runner.seq_length}
+    encoder_seq_length: ${runner.seq_length}
+
+    normalization: rmsnorm
+
+    position_embedding_type: rope
+
+    apply_rope_fusion: True
+    bias_dropout_fusion: False
+    persist_layer_norm: False
+    bias_activation_fusion: False
+    attention_softmax_in_fp32: True
+    batch_p2p_comm: False
+    variable_seq_lengths: True
+    gradient_accumulation_fusion: False
+    moe_token_dispatcher_type: alltoall
+    use_cpu_initialization: False
+
+  optim:
+    optimizer: adam
+    bf16: False
+    fp16: True
+    lr: 2e-05
+    adam_beta1: 0.9
+    adam_beta2: 0.95
+    adam_eps: 1.0e-05
+    min_lr: 2.0e-6
+    weight_decay: 0.05
+    use_distributed_optimizer: True
+    overlap_grad_reduce: False
+    overlap_param_gather: False
+    optimizer_enable_pin: false
+    overlap_param_gather_with_optimizer_step: False
+    clip_grad: 0.8
+    loss_scale: 65536
+
+  lr_sched:
+    lr_warmup_fraction: 0.01
+    lr_warmup_init: 0.0
+    lr_warmup_iters: 0
+    max_lr: 2.0e-5
+    min_lr: 0.0
+    lr_decay_style: constant
+    lr_decay_iters: 10
+
+  tokenizer:
+    tokenizer_model: /mnt/public/hf_models/Qwen2.5-Coder-1.5B
+    use_fast: False
+    trust_remote_code: True
+    padding_side: 'right'
+
+  megatron:
+    ddp_bucket_size: null
+    distributed_backend: nccl # Support 'nccl' and 'gloo'
+    distributed_timeout_minutes: 30
+    ckpt_format: torch
+    use_dist_ckpt: False
+    tp_comm_bootstrap_backend: nccl
+    tp_comm_overlap_cfg: null # tp_comm_overlap_cfg.yaml
+    use_hf_ckpt: False # if true, will transfer hf model to generate megatron checkpoint and use it for training.
+    use_profiler: False # if true, will enable torch profiler when training, pay attention it has influence on performance
+
+    # ckpt_convertor: # config for ckpt convertor
+    #   model: Qwen2.5-Coder-1.5B
+    #   model_type: null # will be set by hf model's config if null
+    #   hf_model_path: ${rollout.model_dir} # path to the hf model
+    #   save_path: ${runner.output_dir}/${runner.experiment_name}/converted_ckpts/actor
+    #   use_gpu_num : 0
+    #   use_gpu_index: null
+    #   process_num: 16 # number of processes to use for checkpointing
+    #   tensor_model_parallel_size: ${actor.model.tensor_model_parallel_size}
+    #   pipeline_model_parallel_size: ${actor.model.pipeline_model_parallel_size}
```

Comment:
what is this section used for? do we have doc to explain how to write this ckpt_convertor section?